packages/ra-ui-materialui/src/input/ArrayInput/SimpleFormIteratorItem.tsx

```diff
@@ -94,24 +96,9 @@ export const SimpleFormIteratorItem = React.forwardRef(
                             inline && SimpleFormIteratorClasses.inline
                         )}
                     >
-                        {Children.map(
-                            children,
-                            (input: ReactElement, index2) => {
-                                if (!isValidElement<any>(input)) {
-                                    return null;
-                                }
-                                const { source, ...inputProps } = input.props;
-                                return cloneElement(input, {
-                                    source: source
-                                        ? `${member}.${source}`
-                                        : member,
-                                    index: source ? undefined : index2,
```

Comment:
in packages/ra-core/src/form/FormDataConsumer.spec.tsx, first test, I see a reference to the `index` prop being injected. Maybe there are other.

Reply:
Nope